htdocs/install/mysql/tables/llx_expeditiondet.sql

```diff
@@ -24,6 +24,7 @@ create table llx_expeditiondet
   rowid             integer AUTO_INCREMENT PRIMARY KEY,
   fk_expedition     integer NOT NULL,
   fk_origin_line    integer,           -- Correspondance de la ligne avec le document d'origine (propal, commande)
+  origin_type		varchar(255),		-- Dans le but de pouvoir créer une expédition sans document lié avec une valeur à product par exemple
```

Comment:
To follow the same name used by reception, can you make a PR with only the database enhancement, that will use add the field

`element_type   varchar(50) DEFAULT 'order' NOT NULL,`